### PR TITLE
HELLO - Is it me you're looking for?

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- Removes a random hardcoded HELLO
- Removes the duplication of a `label` element. By removing the one from `<input>` and surrounding and putting it before the stepper itself. This is better for usability purposes and styling.
- This does introduce a very strange A11Y warning

#### What tests does this PR have?
Checks a label is present when and ID is passed through

#### How can this be tested?
run `npm run docs` click view the stepper component and click on the label (now `number of passengers` rather than `HELLO`) it should enter the input element (which is now NOT read-only by default)

#### Screenshots / Screencast
screen shot 2015-11-25 at 17 00 41

#### What gif best describes how you feel about this work?
HELLO

---

- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by richardstevens** [comment]
Tested this locally and worked as expected, nice touch making the readOnly a prop rather than hardcoded.
:+1:

**Comment by hxpaul** [comment]
Taking review :two:

File: test/components/stepper-test.jsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ describe('Stepper', function() {
       value: 2,
       onChange: sinon.spy(),
       minValue: 1,
-      maxValue: 3
+      maxValue: 3,
+      label: "stepper"
```
Comment (hxpaul) [inline test/components/stepper-test.jsx]: should be single quotes

File: src/components/stepper/stepper.jsx [hunk 2]
```diff
@@ -15,7 +15,8 @@ module.exports = React.createClass({
     incrementDisplayString: React.PropTypes.string,
     decrementDisplayString: React.PropTypes.string,
     id: React.PropTypes.string,
-    label: React.PropTypes.string
+    label: React.PropTypes.string,
+    readOnly: React.PropTypes.bool
```
Comment (hxpaul) [inline src/components/stepper/stepper.jsx]: this new prop is not in the docs - I will add an issue on this repo, maybe we can check this or make the documentation automated soon but for now this one needs to appear in the docs too.

File: docs/src/App.jsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ var React = require('react');
 var HomePage = require('./HomePage.jsx');
 
 var a11y = require('react-a11y');
-//a11y(React, { throw: true, includeSrcNode: true });
+// a11y(React, {includeSrcNode: true });
```
Comment (hxpaul) [inline docs/src/App.jsx]: I see a couple of different messages in the console now when I view the docs is this ok?
```
exports You have `role="button"` but did not define an `onKeyDown` handler. Add it, and have the "Space" key do the same thing as an `onClick` handler.
bundle.js:6541 exports You have `role="button"` but did not define an `onKeyDown` handler. Add it, and have the "Enter" key do the same thing as an `onClick` handler.
bundle.js:6541 exports You have an unlabeled element or control. Add `aria-label` or `aria-labelledby` attribute, or put some text in the element.
```

**Comment by jodiedoubleday** [comment]
Comments addressed.
The react a11y warnings are okay, i think these are solved by fixing the code editor on docs (generally thats not very accessible by nature) we may want to remove this functionality or make it better in the future.

**Comment by hxpaul** [comment]
ok with me now :+1:

**Pull request merged**